test/RPCSession.cpp

```diff
@@ -74,7 +74,14 @@ IPCSocket::IPCSocket(string const& _path): m_path(_path)
 		BOOST_FAIL("Error creating IPC socket object");
 
 	if (connect(m_socket, reinterpret_cast<struct sockaddr const*>(&saun), sizeof(struct sockaddr_un)) < 0)
+	{
+#if defined(_WIN32)
```

Comment:
Isn't this inside the else-part of the global `if defined(_WIN32)`?

Reply:
Oh, that's true.